test/functional/plugin/shada_spec.lua

```diff
@@ -13,14 +13,8 @@ local shada_helpers = require('test.functional.shada.helpers')
 local get_shada_rw = shada_helpers.get_shada_rw
 
 local function reset(shada_file)
-  -- TODO(justinmk): why is this needed?
-  local rtp_value = ('\'%s/runtime\''):format(
-      paths.test_source_path:gsub('\'', '\'\''))
```

Comment:
The TODO was part curiosity about what was intended and whether it's useful for clear() in general.

Reply:
Moved it down with the laststatus.  ok I guess?
`reset` here appears to feature havin a specific fname, which is decent I assumed.

Reply:
I meant the `paths.test_source_path` rtp value.